src/ActionModal/components/ActionModal.vue

```diff
@@ -597,7 +591,11 @@ export default {
       }
     },
     async open() {
-      this.$apollo.skipAll = false
+      if (!this.address) {
+        this.$apollo.skipAll = true
```

Comment:
do we need this?

Reply:
I tried getting rid of `skipAll` altogether, but then a bunch of tests were failing... I can look further into that